env/initialize.sh

```diff
@@ -31,6 +40,10 @@ log_error_message() {
 }
 
 initialize_depot_tools() {
+  if [ -d ./depot_tools -a "${is_forced}" -eq "0" ]; then
```

Comment:
Not familiar with the syntax here - I assume this checks 1) existence of the folder `./depot_tools` and 2) is_forced being equal to 0 - right?

Reply:
Correct - see, e.g., https://tldp.org/LDP/abs/html/comparison-ops.html .